recipes/google-crc32c/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "google-crc32c" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/googleapis/python-crc32c/archive/v{{ version }}.tar.gz
+  sha256: 83144943a727beaffea6d133a1f9e425b46c95d8940ea429091c1c965d9c7ac7
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - libcrc32c
+    - cffi >=1.0.0
+  run:
+    - python
+    - libcrc32c
```

Comment:
`libcrc32c` has a run_exports and is automatically added here with the right pinning.
```suggestion
```